src/MockBot.Api/Models/Chat.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace MockBot.Api.Models;
+
+public class Chat
+{
+    public Guid Id { get; set; } = Guid.NewGuid();
+
+    [Required] public List<Message> Messages { get; set; } = new List<Message>();
```

Comment:
You woudl generally expect ti see annotations on the line above
```suggestion
    [Required]
    public List<Message> Messages { get; set; } = new List<Message>();
```